wikimedia/wikipedia-ios
# Image recommendations - go to Add Image Details

**Phabricator:** https://phabricator.wikimedia.org/T358917

### Notes
* This PR adds the ability to redirect the user to our existing "InsertMediaSettingsViewController" when tapping "Yes" and accepting an image recommendation for an article
* Handling the wikitext is covered in another ticket, so this PR does not go beyond adding info, and does not add any actions to the insert button  
* We still don't have the image displaying here, as handling the image download is yet to be done on WKData
* Note: since both the image title and image description text lengths are unpredictable, I'm setting a hard-coded number of lines for both to limit the size of the header view. With very large texts, it could cause the header to take over most of the view, which I considered to be a bad experience.

### Test Steps
1. On an image recommendation, tap "Yes" on the toolbar
2. Make sure the layout corresponds to Figma and that the "Insert button" is only blue when there is text on the caption fields.
3. Make sure the button is clickable even when greyed out - no action happens but it should be clicked.
4. Go to the Editor and tap Add an image on an article.
5. Make sure it still works as usual.
